docs/hyprctl.1.rst

```diff
@@ -0,0 +1,131 @@
+=======
+hyprctl
+=======
+
+----------------------------------------------------------------
+Utility for controlling parts of Hyprland from a CLI or a script
+----------------------------------------------------------------
+
+:Date: 15 Jul 2022
+:Copyright: Copyright (c) 2022, vaxerski
+:Version: 0.7.1beta
+:Manual section: 1
+:Manual group: hyprctl
+
+SYNOPSIS
+========
+
+``hyprctl`` [(opt)flags] [command] [(opt)args]
+
+DESCRIPTION
+===========
+
+``hyprctl`` is a utility for controlling some parts of the compositor from a CLI or a script.
+If you install with make install, or any package, it should automatically be installed.
```

Comment:
We don't need this line, as people won't have access to this if hyprctl isn't installed. And for those who have it, it's redundant info.